#######################################################################
# This message was automatically generated by the ark_bridge package
#######################################################################
#
# Source message:    .msg
# Creation:          Thu 13 Apr 2017 12:02:49 PM EDT
#
#######################################################################
Header              header
string              child_frame_id
PoseWithCovariance  pose
TwistWithCovariance twist
